index.html

```diff
@@ -767,19 +859,175 @@ <h2><dfn>CaptureController</dfn></h2>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[DisplaySurfaceType]]}} is neither
+                    {{DisplayCaptureSurfaceType/"browser"}} nor {{DisplayCaptureSurfaceType/"window"}},
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[FocusDecisionFinalized]]}} is <code>true</code>,
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If too much time has elapsed since the [=capture-session=] started,
+                    the user agent SHOULD set {{CaptureController/[[FocusDecisionFinalized]]}}
+                    to <code>true</code>. The timespan is left up to the user agent,
+                    but it is recommended that a value of one second be used.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Run the [=finalize focus decision algorithm=].
+                  </p>
+                </li>
+              </ol>
             </dd>
+            <p>
+              The <dfn>finalize focus decision algorithm</dfn> consists of running
+              the following steps:
+            </p>
+            <ol>
+              <li>
+                <p>
+                  If {{CaptureController/[[FocusDecisionFinalized]]}} is <code>true</code>,
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  Set {{CaptureController/[[FocusDecisionFinalized]]}} to <code>true</code>.
+                </p>
+              </li>
+              <li>
+                <p>
+                  If {{CaptureController/[[InitialFocusLost]]}} is <code>true</code>,
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  If {{CaptureController/[[DisplaySurfaceType]]}} is neither
+                  {{DisplayCaptureSurfaceType/"browser"}} nor {{DisplayCaptureSurfaceType/"window"}},
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  If {{CaptureController/[[FocusBehavior]]}} is NOT
+                  {{CaptureStartFocusBehavior/"focus-captured-surface"}},
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  Run the following steps in parallel:
+                </p>
+              </li>
+              <ol>
+                <li>
+                  <p>
+                    Focus the [=display surface=] referred to by {{CaptureController/[[Source]]}}.
```

Comment:
Is there a reference for the "Focus" action? (probably an HTML5 one)

Reply:
The display surface may be a non-HTML window, so I doubt it.

Once we flesh out details more (as I suggested in https://github.com/w3c/mediacapture-screen-share/pull/240#discussion_r991713555), the [system focus](https://html.spec.whatwg.org/multipage/interaction.html#tlbc-system-focus) in HTML might be something we could use when we're capturing `"browser"` surfaces (assuming we mean to give the document focus as well, and not just the browser window).